src/irmin-pack/unix/chunked_suffix.ml

```diff
@@ -35,12 +35,23 @@ module Make (Io : Io.S) (Errs : Io_errors.S with module Io = Io) = struct
     | `Inconsistent_store
     | `Read_out_of_bounds ]
 
+  type add_new_error =
+    [ Io.open_error
+    | `Closed
+    | `Invalid_argument
+    | `Inconsistent_store
+    | `Read_out_of_bounds
```

Comment:
These could be replaced with `open_error`

Reply:
Indeed, thanks